### PR TITLE
Release notes for Tizen studio 3.3 release

### PR DESCRIPTION
**Pull request by vivek-ellur**
Signed-off-by: Vivek Ellur <vivek.ellur@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/tizen-studio/release-notes/3-3-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+-   Release Date: July 26, 2019
+
+
+## IDE and Tools
+
+
+### Fixed Bugs
+-   Device Manager
+	-   Fixed the issue with logs not shown for Raspberry Pi board.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-3-release-notes.md]: To maintain consistency with other Release Notes (has/have been), please rephrase to:
The issue of logs not being displayed for the Raspberry Pi board has been fixed.

Please verify.

File: docs/application/tizen-studio/release-notes/3-3-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+-   IDE
+	-   Fixed the log view to work with TV devices in Tizen IDE.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-3-release-notes.md]: To maintain consistency with other Release Notes (has/have been), please rephrase to:
The issue of the log view for TV devices in the Tizen IDE has been fixed.

Please verify whether the technical meaning is altered.

File: docs/application/tizen-studio/release-notes/3-3-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	-   App signing command incorrectly asking for Author and Distributor passwords has been fixed.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-3-release-notes.md]: To maintain consistency with other Release Notes (has/have been), please rephrase to:
The issue of app signing command incorrectly asking for the author and the distributor passwords has been fixed.
Please verify.

File: docs/application/tizen-studio/release-notes/3-3-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+	-   Running an application on emulator now launches the emulator if not in launched state already.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-3-release-notes.md]: Can we rephrase as:
Running an application on the emulator now launches the emulator automatically if the emulator is not in the launched state.

Please verify.

File: docs/application/tizen-studio/release-notes/3-3-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+
+### Fixed Bugs
+-   Device Manager
+	-   The issue of logs not being displayed for the Raspberry Pi board has been fixed.
+-   IDE
+	-   The issue of the log view for TV devices in the Tizen IDE has been fixed.
+	-   The issue of the app signing command incorrectly asking for the author and the distributor passwords has been fixed.
+	-   Running an application on the emulator now launches the emulator automatically if the emulator is not in the launched state.
+	-   The launch_screen element to platform schema for the web applications has been added.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-3-release-notes.md]: One more question, "launch_screen" element is part of the code?
If it is, then can we use the code tag to emphasize?

The `launch_screen` element...

Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-3-release-notes.md]: it is part of schema.

File: docs/application/tizen-studio/release-notes/3-3-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+# Tizen Studio 3.3 Release Notes
+
+-   Release Date: July 26, 2019
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/3-3-release-notes.md]: The expected target date is July 24.
So would you please change to "July 24, 2019"?

Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-3-release-notes.md]: Updated the date.

**Pull request merged**